common/source/docs/common-cuav-x7plus-overview.rst

```diff
@@ -0,0 +1,182 @@
+|                     |ICM20689    |ICM20689         |BMI088            |ICM42688-P        |
+|                     +------------+-----------------+------------------+------------------+
+|                     |ICM20649    |ICM20689         |ICM20649          |ICM20689          |
+|                     +------------+-----------------+------------------+------------------+
+|                     |                              MS5611*2                              |
+|                     +------------+-----------------+------------------+------------------+
+|                     |                              RM3100                                |
++---------------------+------------+-----------------+------------------+------------------+
+|PWM  outputs         |14          |14               |14                |14                | 
++---------------------+------------+-----------------+------------------+------------------+
+|DBSTOT channel       |6(M9~M14)   |14(M1~M14 )      |6(M9~M14)         |14(M1~M14 )       |
++---------------------+------------+-----------------+------------------+------------------+
+|Servo voltage monitor|NO          | YES(9.9V max)   |NO                | YES(9.9V max)    | 
++---------------------+------------+-----------------+------------------+------------------+
+|Operating Voltage    |                      4.3~5.4V （USB:4.75~5.25V)                    |
++---------------------+------------+-----------------+------------------+------------------+
+|Power monitor        |          2(1 analog power monitor, 1dronecan power monitor)        |
++---------------------+------------+-----------------+------------------+------------------+
+
+Interfaces
+==========
+
+   -  14 PWM servo outputs（12 support DShot）
+   -  Analog/ PWM RSSI input
+   -  2 GPS ports(GPS and UART4 ports)
+   -  4 I2C buses(Two external I2C ports)
+   -  2 CAN bus ports
+   -  2 Power ports(Power A is an ADC interface, Power C is a DroneCAN battery interface)
+   -  2 ADC inputs
+   -  1 High Speed (480mb/s)USB port
+
+-  **Other**
+
+   -  Weight: 101g（X7+ Pro：105g）
+   -  Operating temperature: -20 ~ 80°c（Measured value）
+
+Size and Pinouts
+================
+
+.. image:: ../../../images/cuav_autopilot/x7plus/x7_plus_size.png
+    :target: ../_images/cuav_autopilot/x7plus/x7_plus_size.png
+    
+.. image:: ../../../images/cuav_autopilot/x7plus/pinouts_01.png
+    :target: ../_images/cuav_autopilot/x7plus/pinouts_01.png 
+
+.. image:: ../../../images/cuav_autopilot/x7plus/pinouts_02.png
+    :target: ../_images/cuav_autopilot/x7plus/pinouts_02.png    
+
+Default UART Order
+==================
+
+- SERIAL0 = console = USB
+- SERIAL1 = Telemetry1 = USART1
+- SERIAL2 = Telemetry2 = USART6
+- SERIAL3 = GPS1 = USART1
+- SERIAL4 = GPS2 = UART4
+- SERIAL5 = USER = UART8 
+- SERIAL6 = USER = UART7 (DEBUG TX/RX)
+
+Serial protocols can be adjusted to personal preferences.
+
```

Comment:
needs an RC input section stating where to connect, which protocols are supported on that pin (PPM,SBUS,DSM,etc.), and that serial RX like CRSF,FPort, SRLX2, etc need to use a full UART